backup_codecommit.sh

```diff
@@ -23,18 +23,14 @@ git config --global credential.UseHttpPath true
 
 declare -a repos=(`aws codecommit list-repositories | jq -r '.repositories[].repositoryName'`)
 
-for codecommitrepo in "${repos[@]}"
```

Comment:
I put the cloning code itself into a new `backup_repo.sh` script, to make it more easily callable by Parallel.